<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>572</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>561</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>70</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="settings">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>561</width>
      <height>21</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Nazwa</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Miasto</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Województwo</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="mainLabel">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>100</y>
      <width>641</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Program do obsługi API GIOS</string>
    </property>
   </widget>
   <widget class="QLabel" name="stationId">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>80</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Projekt JPO</string>
    </property>
   </widget>
   <widget class="QLabel" name="communeAndProvince">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>210</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="adress">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>150</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>160330 Maksym Górny LAB04</string>
    </property>
   </widget>
   <widget class="QLabel" name="city">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>180</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="province">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>240</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="coords">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>270</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="airIndex">
    <property name="geometry">
     <rect>
      <x>1043</x>
      <y>87</y>
      <width>210</width>
      <height>32</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::RightToLeft</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QListWidget" name="sensorListWidget">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>180</y>
      <width>241</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="sensorTableWidget">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>180</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Rodzaj</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Indeks</string>
     </property>
    </column>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>71</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>Wszystkie</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PM10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SO2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PM25</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>O3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>NO2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C6H6</string>
     </property>
    </item>
   </widget>
   <widget class="QListWidget" name="dataListWidget">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>310</y>
      <width>241</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="folderButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
